pulp_2to3_migration/app/tasks/migrate.py

```diff
@@ -39,7 +39,7 @@ def migrate_from_pulp2(migration_plan_pk, dry_run=False):
 
     # TODO: Migration Plan parsing and validation
     # For now, the list of plugins to migrate is hard-coded.
-    plugins_to_migrate = ['iso']
+    plugins_to_migrate = ['iso', 'docker']
```

Comment:
I don't know if we want to commit both or not. Not everyone has pulp 2 docker installed in the dev env.
At the same time it should go away with @dralley 's PR anyway. Up to you, whether to keep it or not.

Reply:
i hardcoded it so it can be tested at least. i will leave this to @dralley once he will rebase it will be removed